src/core_atmosphere/Registry.xml

```diff
@@ -1891,6 +1966,11 @@
                      units="s"
                      description="Relaxation time to initial values in 1-d OML"
                      possible_values="Non-negative real values"/>
+
+                <nml_option name="config_lightning_flash" type="logical" default_value="false" in_defaults="false"
+                     units="-"
+                     description="Turns on/off the calculation of storm parameters to compute lightning flash rates"
+                     possible_values=".true. or. false"/>
```

Comment:
I think the lightning flash rate diagnostics were something we added to the HWT runs, but these should not be included in the changes for Noah.